src/components/interfaces/HMI_API.xml

```diff
@@ -7517,7 +7517,7 @@
 
 </interface>
 
-<interface name="AppService" version="1.0.0" date="2019-03-18">
+<interface name="AppService" version="1.1.0" date="2021-04-14">
```

Comment:
I don't see any changes in the AppService interface, should it be version bumped?